tmt/convert.py

```diff
@@ -156,11 +157,11 @@ def read_nitrate(beaker_task, common_data):
         data = dict()
         echo("test case found '{0}'.".format(testcase.identifier))
         # Test identifier
-        data['tcms'] = testcase.identifier
+        data['extra-tcms'] = testcase.identifier
```

Comment:
This key has to also be changed in cli.py where is it used or it tracebacks
line 467 in cli.py `testcase_path = os.path.join(path, str(testcase['tcms']) + '.fmf')`

Reply:
Ah, good catch!